test/csuite/timestamp_abort/main.c

```diff
@@ -451,6 +453,12 @@ run_workload(uint32_t nth)
 	if (compat)
 		strcat(envconf, ENV_CONFIG_COMPAT);
 
+	if (nth > SESSION_MAX) {
```

Comment:
I don't think this needs to be conditional - just configure enough sessions so that the maximum thread count won't run out. It's also not going to be OK to configure 99 threads with this logic.